ronin/drive-storage.rst

```diff
@@ -0,0 +1,24 @@
+.. _drive-storage:
+
+Drive Storage
+=======================================
+
+Drive storage or block storage as it's often referred as is the storage attached directly to your :term:`instances<instance>` within Ronin.
+These are most commonly the "Root Drive" however Ronin gives you the option to create your own additional storage to attach and move between instances.
```

Comment:
Link to the relevant RONIN documentation so people can learn how to do this

Reply:
https://blog.ronin.cloud/storage-summary-help/
This?

We already link to a few docs from here https://docs.rcc.shef.ac.uk/ronin/index.html but It looks like it's worth adding.

Reply:
Er, there's a few pages:

- [Managing storage drives in RONIN](https://blog.ronin.cloud/storage-help/)
- [Everything You Never Needed to Know About Drives, File Systems and Storage](https://blog.ronin.cloud/volumes-filesystems-storage/#em-classfas-fa-rocketem-resizing-drives)